### ISSUE TITLE
[React Native] - Form component causes 'Expected a component class, got form'

### ISSUE THREAD

**Issue by bgriffith** [davidkpiano/react-redux-form#166]
Hey @davidkpiano - awesome library!

I'm getting a similar error to #116 if I don't specify a `component` attribute for the `Form` element, which I guess is down to it missing a default value (i.e `View`) for that attribute? 

> Expected a component class, got Form.

Either way, it disappeares after I plug my own custom component in (now that I understand RRF better it makes sense) but just to highlight that it wasn't initially obvious that it was required from the docs.

:+1:

**Comment by davidkpiano** [comment]
Thanks for the report, @bgriffith! Yeah, this is an interesting use-case because React-Native doesn't have the notion of a "form" component, but I understand why you might still want to use `<Form>` in React-Native. I'll clarify this in the docs.

**Comment by davidkpiano** [comment]
I published an addition for this as `0.11.1`: https://github.com/davidkpiano/react-redux-form/commit/5cf2044a68a93da415dfb427251412c3f1cc2f8b

There is now a more succinct way to grab the `<Form>` as a rendered `<View>`:

```js
import { NativeForm as Form } from 'react-redux-form/lib/native';

// render():
<Form model="...">
```

**Comment by bgriffith** [comment]
Awesome, thanks!
